src/pages/community/members.js

```diff
@@ -14,11 +14,11 @@ import Maintainers from "../../sections/Community/Members-grid/Maintainers";
 import Meshmate from "../../sections/Community/Members-grid/Meshmate";
 
 const options = [
-  { label: "All Members", value: "all" },
-  { label: "Active Members", value: "active" },
-  { label: "Inactive Members", value: "inactive" },
-  { label: "Maintainers", value: "maintainers" },
-  { label: "MeshMates", value: "meshmates" }
+  { label: "All Members", value: "all", color: "#0bb39f", isFixed: true},
```

Comment:
It'd be great if we could use the properties from src/theme/app/themeStyles.js here.